test/cli/page_width_test.dart

```diff
@@ -12,6 +12,54 @@ import '../utils.dart';
 void main() {
   compileFormatter();
 
+  group('--page-width', () {
+    test('must be a number', () async {
+      var process = await runFormatter(['--page-width=12.34']);
+      await process.shouldExit(64);
+    });
+
+    test('must be an integer', () async {
+      var process = await runFormatter(['--page-width=notint']);
+      await process.shouldExit(64);
+    });
```

Comment:
[nit] The test titles and conditions look swapped

```suggestion
    test('must be a number', () async {
      var process = await runFormatter(['--page-width=notint']);
      await process.shouldExit(64);
    });

    test('must be an integer', () async {
      var process = await runFormatter(['--page-width=12.34']);
      await process.shouldExit(64);
    });
```